apps/plasmo/src/components/Calendar/CalendarMonthYear.tsx

```
@@ -0,0 +1,179 @@
+    "10": "Magh",
+    "11": "Falgun",
+    "12": "Chaitra"
+  }
+
+  const monthOptions = Object.entries(nepaliMonths).map(([value, label]) => ({
+    value,
+    label: `${label} (${value})`
+  }))
+
+  const handlePreviousMonth = () => {
+    const currentMonthIndex = parseInt(selectedMonth)
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)